packages/type-css-modules/.gitignore

```diff
@@ -1,5 +1,5 @@
 # compiled output
-# /dist/
+/dist/
```

Comment:
In #73, I had committed `dist` as a workaround, as I didn't know back then how a codemod can be written in TypeScript, then its code compiled to JavaScript before publishing it to `npm`.